spec/models/task_spec.rb

```diff
@@ -1,5 +1,43 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
+    it 'is valid with all attributes' do
+      user = User.create(email: "user@sample.com", password: "password")
+      task = user.tasks.new(title: "aaa", content: "aaa", status: :todo)
+      expect(task.valid?).to be(true)
+      expect(task.errors).to be_empty
+    end
+
+    it 'is invalid without title' do
+      user = User.create(email: "user@sample.com", password: "password")
+      task = user.tasks.new(title: "", content: "aaa", status: :todo)
+      expect(task.valid?).to be(false)
+      expect(task.errors).not_to be_empty
+    end
+
+    it 'is invalid without status' do
+      user = User.create(email: "user@sample.com", password: "password")
+      task = user.tasks.new(title: "aaa", content: "aaa", status: nil)
+      expect(task.valid?).to be(false)
+      expect(task.errors).not_to be_empty
+    end
+
+    it 'is invalid with a duplicate title' do
+      user = User.create(email: "user@sample.com", password: "password")
+      task1 = user.tasks.new(title: "aaa", content: "aaa", status: :todo).save
+      task2 = user.tasks.new(title: "aaa", content: "aaa", status: :todo)
+      expect(task2.valid?).to be(false)
+      expect(task2.errors).not_to be_empty
+    end
+
+    it 'is valid with another title' do
+      user = User.create(email: "user@sample.com", password: "password")
+      task1 = user.tasks.new(title: "aaa", content: "aaa", status: :todo).save
+      task2 = user.tasks.new(title: "bbb", content: "aaa", status: :todo)
```

Comment:
変数の役割や内容が分かる様な名前をつけた方が良いと思います！
```suggestion
      task_with_another_title = user.tasks.new(title: "bbb", content: "aaa", status: :todo)
```

Reply:
419daf1